utils/hct/pkgesbuild/format-status-email.ps1

```diff
@@ -0,0 +1,74 @@
+﻿param(
+    [parameter(Mandatory=$true)]  [System.IO.FileInfo]$BuildLogs,
+    [parameter(Mandatory=$false)] [System.IO.FileInfo]$EmailBodyFile = "email-body.html",
+    [parameter(Mandatory=$false)] [string[]]$StatusItems = @("Build", "Tests"),
+    [parameter(Mandatory=$false)] [string[]]$StatusItemsCanFail = @("Tests")
+)
+
+$platforms = "x64", "x86", "arm64"
+$flavors = "Debug", "Release"
+
+$overall_status =  "Succeeded"
+
+function Get-TDStyle {
+    param([string]$status)
+
+    switch ($status) {
+        "Succeeded"           { return "background-color:#BBFFBB" }
+        "SucceededWithIssues" { return "background-color:#FFEE88" }
+        "Failed"              { return "background-color:#FF8888" }
+        default               { return "" }
+    }
+}
+
+$status_tables=""
```

Comment:
I'd just comment that this outputs an html table... or several? With per platform+flavor build status.

Reply:
Right now just the default one table for build status and one for tests status, listing all flavors/platforms. If we want to have more test groups it should be easy to add by setting the StatusItems argument on the script.